pages/profile.tsx

```diff
@@ -1,37 +1,63 @@
 import React from "react";
-import { Container, ProgressBar } from "react-bootstrap";
+import { Button, Container, ProgressBar, Image } from "react-bootstrap";
+import Link from "next/link";
+import { FaExpand } from "react-icons/fa";
+import { useUser } from "../lib/hooks/auth";
 
 export default function Profile() {
+  const { user } = useUser();
   return (
-    <Container className="mt-5 text-center d-grid gap-2">
-      <h1>Statistics</h1>
-      <table className="Statistics">
-        <thead>
-          <tr>
-            <th scope="col">12</th>
-            <th scope="col">92</th>
-            <th scope="col">6</th>
-            <th scope="col">6</th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr>
-            <td scope="row">Played</td>
-            <td>Win %</td>
-            <td>Current Streak</td>
-            <td>Maximum Streak</td>
-          </tr>
-        </tbody>
-      </table>
-      <Container className="mt-5 text-center d-grid gap-2">
-        <h1> Guess Distribution </h1>
-        <ProgressBar now={100 / 4} label={`1`} variant="success" />
-        <ProgressBar now={0} label={`0`} variant="secondary" />
-        <ProgressBar now={100 / 4} label={`1`} variant="secondary" />
-        <ProgressBar now={100 / 1} label={`4`} variant="secondary" />
-        <ProgressBar now={100 / 4} label={`1`} variant="secondary" />
-        <ProgressBar now={100 / 4} label={`1`} variant="secondary" />
-      </Container>
-    </Container>
+    <div className="container">
+      <div className="row">
+        <div className="col-sm-6 text-center">
```

Comment:
Consider using the React Bootstrap versions of these instead of the vanilla bootstrap versions:
`<Container>` and `<Row>` and `<Col>` (full documentation: https://react-bootstrap.github.io/)